### DISCUSSION TITLE
github.dev: Unable to authenticate your session

### DISCUSSION THREAD

**Discussion by NoCrypt** [community/community#43020] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi! I want to open my repo on github.dev, but everytime I open it, it just redirect me a few times back and forth from github.dev to github.com then this message shows up:
![ShareX_23-01-01 094155](https://user-images.githubusercontent.com/57245077/210159585-834951b9-5180-40a8-8190-c3570ad69006.png)

Here are things that I have tried:
- Disabled Ad-blocker
- Log out and Login

**Comment by jfullstackdev** [comment]
yup even in Codespaces there is the problem, though there is no report in GitHub Status

![image](https://user-images.githubusercontent.com/47092464/210161100-f3281f71-f38c-4122-acc0-fcc5ad1a216e.png)

**Reply by rajivmehtaflex** [reply]
I'm facing same issue.

**Reply by pattar314** [reply]
Same issue

**Reply by AkaShark** [reply]
+1

**Reply by shinchuma** [reply]
+1

**Reply by tshion** [reply]
I'm facing same issue from 2023-01-01T10:01:00+09:00.

* Google Chrome 108.0.5359.124（Official Build） （x86_64）
* macOS Monterey 12.6

**Reply by svenmalvik** [reply]
Doesn't work for me either on Chrome Version 108.0.5359.125 (Official Build) (64-bit) and Windows 11

**Reply by christianbraeunlich** [reply]
+1

**Comment by RexRowan** [comment]
Same

**Comment by cyfung1031** [comment]
I tried to clear the site cache for both github.dev and github.com using https://thegeekpage.com/clear-site-data-for-a-particular-website/

not working

Edit: 2022/01/02 fixed

**Reply by mike-luabase** [reply]
Incognito browser also doesn't work

**Reply by cyfung1031** [reply]
fixed now :)

**Comment by sanjaiyan-dev** [comment]
Any solutions please ? 
Facing same problem :(

**Comment by Loc-Tran-2111** [comment]
Same problem too :(

**Comment by fleizean** [comment]
same problem :((((

**Comment by Plasma** [comment]
Codespaces haven’t been loading for me either

**Comment by uluumbch** [comment]
hmm, at least i'm not alone LOL

**Comment by radiumatic** [comment]
Same problem.

**Comment by MananSahniAdpushup** [comment]
Same problem for me too.

**Reply by MananSahniAdpushup** [reply]
Working for me now.

**Comment by ChienLeeVer** [comment]
i have the same problem

**Comment by vidhyasagar-sb** [comment]
I'm facing the same issue

**Comment by tfx2001** [comment]
Same here. And I also try codespace, it didn't work either.

**Comment by pl4nty** [comment]
/keychain-keys is returning empty responses for me, causing a cascading error

**Comment by minhtri1401** [comment]
same here

**Comment by Emonsheikh3666** [comment]
I tried using codespaces and github.dev but both of them kept redirecting and eventually saying that It can't authenticate my session

**Comment by antonioTOM** [comment]
I have the same problem for github.dev and codespace.

**Comment by Playback80** [comment]
> ### เลือกหัวข้อหัวข้อ
> บั๊ก
> 
> ### ร่างกาย
> สวัสดี! ฉันต้องการเปิด repo บน github.dev แต่ทุกครั้งที่ฉันเปิด มันแค่เปลี่ยนเส้นทางฉันไปมาสองสามครั้งจาก github.dev ไปที่ github.com จากนั้นข้อความนี้ก็ปรากฏขึ้น: ![ShareX_23-01-01 094155](https://user-images.githubusercontent.com/57245077/210159585-834951b9-5180-40a8-8190-c3570ad69006.png)
> 
> นี่คือสิ่งที่ฉันได้ลอง:
> 
> * ปิดการใช้งาน Ad-blocker
> * ออกจากระบบและเข้าสู่ระบบ

**Comment by navilg** [comment]
Same issue with me on Firefox 108.0.1

**Comment by sanskaromar** [comment]
I am facing same issue.

I tried:
- Logging out and Logging In
- Cache clear
- Incognito Mode
- Changing browsers from chrome to firefox 
- disabling adblockers

**Comment by nanjekyejoannah** [comment]
It seems to be a general problem. Facing same issue too maybe github support should investigate.

**Reply by iencodev** [reply]
I have the same issue right now, in my case using Safari on iPadOS

**Comment by antonioTOM** [comment]
I don't know you guys, but my codespace is working again, check if it's the same for you all

**Reply by iencodev** [reply]
Now working well for me too, thanks for the heads up @antonioTOM

**Reply by christianbraeunlich** [reply]
Confirmed 👍 Thanks @antonioTOM for the update on this one

**Comment by NoCrypt** [comment, marked as answer]
Seems like the issue have been fixed.

**Comment by jfullstackdev** [comment]
yup, it's already fixed by now

**Comment by 4msar** [comment]
![image](https://user-images.githubusercontent.com/17962782/212283354-34996f3e-5c60-4dc7-9846-f7129d045929.png)

I'm also facing this issue since few days, I can't open the github.dev neither codespace.
Still it's not working.

@github-staff

**Comment by hacatu** [comment]
This issue happens for me when I go to github.dev on firefox in a vm, but not when I go to github.dev on firefox outside a vm.

Very embarassingly, github have DISABLED ISSUE REPORTING for the github/dev repository, so frankly I doubt this will ever get fixed until and unless an accidental change fixes it, like whatever accidental change initially broke it :/
